website/pages/docs/configuration/server.mdx

```diff
@@ -141,6 +141,14 @@ server {
   features and is typically not required as the agent internally knows the
   latest version, but may be useful in some upgrade scenarios.
 
+- `raft_multiplier` `(int: 1)` - An integer multiplier used by Nomad servers to
+  scale key Raft timing parameters. Omitting this value or setting it to 0 uses
+  default timing described below. Lower values are used to tighten timing and
```

Comment:
"described below" - we don't describe it below.

I don't think this documentation from Consul works well for us because they set their `raft_multiplier` to 5 while we set it to the minimum value of 1. There's no way to adjust Nomad to be _more sensitive_ to faults.

If it's ok that Nomad cannot be tuned to be more sensitive (we're really locking ourselves in for the long term with this design), and I think that's ok, then maybe the docs should read:

> `raft_multiplier` `(int: 1)` - An integer multiplier used by Nomad servers to scale key Raft timing parameters similar to [Consul's `raft_multiplier`](https://www.consul.io/docs/agent/options#raft_multiplier). Unlike Consul, Nomad defaults to a multiplier of `1`, so timings can only be increased. Increasing the timings makes leader election less likely during periods of networking issues or resource starvation. Since leader elections pause Nomad's normal work, it may be beneficial for slow or unreliable networks to wait longer before electing a new leader. The tradeoff when raising this value is that during network partitions or other events (server crash) where a leader is lost, Nomad will not elect a new leader for a longer period of time than the default. The [`nomad.nomad.leader.barrier'` metric](/docs/telemetry/metrics/) is a good indicator of how often leader elections occur.

Reply:
Thanks for catching this.  I'll follow up.  To clarify, Nomad's raft_multiplier=1 matches Consul's raft_multiplier=1 - i.e. nomad's minimum is consul's minimum.  When Consul introduced raft_multipler flag in 0.7, they additionally changed the default to make it looser to accomodate dev instances - here, I didn't change the default, hence it stays at 1 - the equivalent of pre-0.12 values.